instrumentation/kafka-clients/src/main/java/brave/kafka/clients/TracingProducer.java

```diff
@@ -130,6 +131,10 @@
     delegate.close(timeout, unit);
   }
 
+  @Override public void close(Duration duration) {
```

Comment:
guessing this was prior java 8 bytecode anyway, ya? I think you might want to remove Override though. 

See "Tracing multiple versions of an interface with one instrumentation class" here https://cwiki.apache.org/confluence/display/ZIPKIN/Java++Instrumentation+Techniques

it would be cool to have an invoker test like grpc to prove older versions still work

Reply:
True, I forgot this. 

We have and `it` to test this module against `kafka-clients:1.0.0` but is not triggering an exception when `@Override`. Any clue on how to test this?

Reply:
maybe it is because we aren't compiling against that method in the IT? it is a good question!